Replicator/Pusher.cc

```diff
@@ -72,6 +72,32 @@ namespace litecore { namespace repl {
     }
 
 
+    bool Pusher::isDocumentAllowed(C4Document* doc) const {
+        if(!isDocumentIDAllowed(doc->docID)) {
+            return false;
+        }
+
+        if(!_options.pushFilter) {
+            return true;
+        }
+
+        const auto body = doc->selectedRev.body;
```

Comment:
For consistency, call `DBAccess::getDocRoot(doc)` instead of doing the work yourself.

Reply:
Did it myself :)